humility-core/src/hubris.rs

```diff
@@ -274,12 +274,43 @@ impl HubrisSensorKind {
     }
 }
 
+//
+// This is the Hubris definition
+//
+#[derive(Debug, Deserialize)]
+pub enum FlashProgram {
+    PyOcd(Vec<FlashArgument>),
+    OpenOcd(FlashProgramConfig),
+}
+
+#[derive(Debug, Deserialize)]
+pub enum FlashProgramConfig {
+    Path(Vec<String>),
+    Payload(String),
+}
+
+#[derive(Debug, Deserialize)]
+pub enum FlashArgument {
+    Direct(String),
+    Payload,
+    FormattedPayload(String, String),
+    Config,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct HubrisRawFlash {
```

Comment:
Changing the name here (from `FlashConfig`) is a little confusing – maybe `HubrisFlashConfig` instead?

`HubrisRawFlash` makes me think of the _contents_ of flash, rather than a configuration.